app/reducers/userState.ts

```diff
@@ -39,3 +50,7 @@ export function selectIsKnownUser(state: IUserState): boolean {
 export function selectLoading(state: IUserState): boolean {
   return state.loading;
 }
+
+export function selectDerivationPath(state: IUserState): string {
```

Comment:
this was used anywhere?

Reply:
Not yet - it would be part of choosing account for user.